### ISSUE TITLE
FL: Pull in and display operationalHoursSpecialInstructions for Vet Centers

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#18911]
## Issue Description
VAST administrators have included a new `operationalHoursSpecialInstructions` field that is now in Lighthouse.

![operational status](https://user-images.githubusercontent.com/55411834/105556756-e5e26980-5cc7-11eb-85dd-56ac73acca56.jpg)

There are 4 possible options that could be concatenated from VAST:
- Administrative hours are Monday-Friday 8:00 a.m. to 4:30 p.m.
- Expanded or Nontraditional hours are available for some services on a routine and or requested basis. Please call our main phone number for details.
- Vet Center after hours assistance is available by calling 1-877-WAR-VETS (1-877-927-8387).
- Vet Center hours are dependent upon outreach assignments. Please call our main phone number for details.

Field may contain one or more of the options above. If more than one is sent, all will be displayed. 

<detail>
<summary> Similar contents already baked into the Vet Center detail page front end </summary>

![image](https://user-images.githubusercontent.com/55411834/105557726-1e834280-5cca-11eb-8a1a-4fce7f3a9cec.png)

</details>

NOTE: Current language in VAST is not Veteran friendly but we will work with Jessica Schiefer to improve and standardize the wording for Vet Centers. Changing it in VAST will automatically update the front end. 

This impacts VHA only : VAMCs and Vet Centers
 
---
## Tasks
- [ ] Keep hardcoded language for now and put this on a flipper
- [ ] Pull in `operationalHoursSpecialInstructions` field language 
- [ ] Replace existing detail page language with language from field 

## Acceptance Criteria
- [ ] Language displayed on Vet Center detail page matches language from `operationalHoursSpecialInstructions` field 
- [ ] If more than one option is sent, all display. 
- [ ] If null value - nothing displays
- [ ] Existing functionality does not change in Production until flag changes
- [ ] Ensure formatting of value(s) is correct (no pipes, etc)

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mmiddaugh** [comment]
If data has pipes - will need to ask Lighthouse to clean data (rather than cleaning it on our side)

**Comment by davidconlon** [comment]
@davidconlon needs to work with Jessica Schieffer about this first for Vet Centers.

**Comment by mmiddaugh** [comment]
Blocked by need to scrub data formatting (pipes)

**Comment by mmiddaugh** [comment]
Closing as no longer relevant
